Documentation/devel/release.md

```diff
@@ -4,40 +4,42 @@ How to perform a release of rkt.
 - Update the [roadmap](https://github.com/coreos/rkt/blob/master/ROADMAP.md) to remove the release you're performing, if necessary
 - Branch from the latest master, make sure your git status is clean
-- Ensure the build is clean! `make` should work, `make check` should pass, functional + integration tests on CI should be green
+- Ensure the build is clean!
+  - `./autogen.sh && ./configure --enable-functional-tests && make && make check` should work
+  - Integration tests on CI should be green
 - Update the [release notes](https://github.com/coreos/rkt/blob/master/CHANGELOG.md). Try to capture most of the salient changes since the last release, but don't go into unnecessary detail (better to link/reference the documentation wherever possible).
 
 The rkt version is [hardcoded in the repository](https://github.com/coreos/rkt/blob/master/version/version.go#L17), so the first thing to do is bump it:
-- Run `scripts/bump-release v0.1.2`. This should generate two commits: a bump to the actual release (e.g. v0.1.2), and then a bump to the release+git (e.g. v0.1.2+git). The actual release version should only exist in a single commit!
+- Run `scripts/bump-release v0.7.0`. This should generate two commits: a bump to the actual release (e.g. v0.7.0), and then a bump to the release+git (e.g. v0.7.0+git). The actual release version should only exist in a single commit!
 - Sanity check what the script did with `git diff HEAD^^` or similar. As well as changing the actual version, it also attempts to fix a bunch of references in the documentation etc.
-- Fix the commit `HEAD^^` so that the version in `stage1/rootfs/aggregate/aci-manifest` is correct.
+- Fix the commit `HEAD^^` so that the version in `stage1/aci/aci-manifest` is correct.
 - If the script didn't work, yell at the author and/or fix it. It can almost certainly be improved.
 - File a PR and get a review from another [MAINTAINER](https://github.com/coreos/rkt/blob/master/MAINTAINERS). This is useful to a) sanity check the diff, and b) be very explicit/public that a release is happening
 - Ensure the CI on the release PR is green!
 
 After merging and going back to master branch, we check out the release version and tag it:
 - `git checkout HEAD^` should work (or `git checkout HEAD^2~`? git how does it work); sanity check version/version.go after doing this
-- Build the release with `./build` with the default build options (don't define any environment variables `RKT_STAGE1_*`, ); we'll use this in a minute. Sanity check `bin/rkt version`
-- Add a signed tag: `git tag -s v0.1.2`. (We previously used tags for release notes, but now we store them in CHANGELOG.md, so a short tag with the release name is fine).
+- Build the release with `./configure && make`, we'll use this in a minute. Sanity check `bin/rkt version`
```

Comment:
The build command should be: `./autogen.sh && ./configure && make`

Since configure.ac was changed with the new version, autogen.sh is needed to select the correct build directory.